src/watchdog/observers/kqueue.py

```diff
@@ -126,6 +126,10 @@
     | select.KQ_NOTE_REVOKE
 )
 
+
+def absolute_path(path):
+    return Path(path).resolve()
```

Comment:
About that change, I am not sure enough. `resolve()` is not yet completely reliable. See that [thread](https://discuss.python.org/t/have-a-realpath-classmethod-in-pathlib-path/4692/15?u=tiger-222) for instance. So, two things:
- either we go as-is and see if it breaks somewhere
- either we go with `os.path.abspath(str(path))`

I do not have a strong opinion right now.

Reply:
[`pathtools.absolute_path`](https://github.com/gorakhargosh/pathtools/blob/a3522fc61b00ee2d992ca375c600513bfb9020e9/tests/run_tests.py#L9) is doing `abspath(normpath(path))`.

I'm also not sure what's the safest option right now. I'm fine with changing this line in any way you see fit.

Reply:
Well we can go as-is.